test/sqlgen.js

```diff
@@ -161,7 +178,7 @@ test.testSync('Select all order offset', (test, { builder, params }) => {
     .orderBy('f1')
     .offset(10);
   const query = builder.build();
-  test.strictSame(query, 'SELECT * FROM "table" ORDER BY "f1" ASC OFFSET $1');
+  test.strictSame(query, 'SELECT * FROM "table" ORDER BY "f1" OFFSET $1');
```

Comment:
I'd prefer to always have order specified explicitly.

Reply:
Nobody use ASC in SQL, it's by default in all DBMS

Reply:
I wouldn't be so sure that no one uses the default order specifier ASC, but regardless, here we just wanted to explicitly specify it to increase the readability of the resulting query, since it was supposed to be human-readable.